### DISCUSSION TITLE
Disable tasks on runtime

### DISCUSSION THREAD

**Discussion by konradmichalik** [deployphp/deployer#3491] [General]
Hej there,
is it possible to disable tasks on runtime? 
So for example something like this:
```php
task('deploy:test', function() {
  task('deploy:success')->disable();
});
```
Or one the other hand, is it possible to disable tasks dependent on the host?
Thanks

**Comment by antonmedv** [comment]
You can use selector for this.

**Reply by konradmichalik** [reply]
Thanks for the hint, this is helping so far.
